logging/appender.go

```diff
@@ -36,6 +37,26 @@ func NewWriterAppender(writer io.Writer) ConsoleAppender {
 	return ConsoleAppender{writer}
 }
 
+// NewFileAppender will create an Appender that writes output to a log file. Log rotation will be
+// enabled such that restarts of the viam-server with the same filename will move the old file out
+// of the way. The `io.Closer`
```

Comment:
```suggestion
// of the way. The `io.Closer` can be used to eventually close the opened log file.
```

Reply:
Also clever way to minimize the methods devs can call on the `*lumberjack.Logger`.

Reply:
Thanks for spotting the unfinished sentence -- added by hand